ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -145,14 +145,37 @@ tests:
         IPv6=yes
         IPv4=yes
     workflow: baremetalds-assisted
-- as: e2e-metal-assisted-multi-version-suite
+- as: e2e-metal-assisted-4.6
```

Comment:
we already have something similar upwards (``e2e-metal-assisted-ocp46``)
but we should definitely change the names to your new scheme